ssv/spectest/tests/runner/postconsensus/invalid_msg_sig.go

```diff
@@ -9,10 +9,9 @@ import (
 	"github.com/bloxapp/ssv-spec/types/testingutils"
 )
 
-// InvalidMessageSignature tests PartialSignatureMessage signature invalid
+// InvalidMessageSignature tests PartialSignatureMessage signature invalid. No error is generated since the SignedPartialSignatureMessage.Signature is no longer checked
```

Comment:
```suggestion
// InvalidMessageSignature tests PartialSignatureMessage signature invalid. No error is generated since the SignedPartialSignatureMessage. Signature is no longer checked. Wait for quorum before verifying.
```

Reply:
The "no space" is intended. I wanted it to be: "SignedPartialSignatureMessage. Signature"